src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets

```diff
@@ -7,4 +7,8 @@
     <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata" />
```

Comment:
Can we switch the backslashes in this file to forward slashes? The reason is that in Linux backslashes are permited as a file name, so sometimes this has unexpected results.